server/views/macros/assessmentInformation.njk

```diff
@@ -19,6 +19,9 @@
               <tr class="govuk-table__row">
                 <th class="govuk-table__header" scope="row">{{ questionAnswer.questionTitle }}</th>
                 <td class="govuk-table__cell">{{ questionAnswer.answer }}</td>
+                <td class="govuk-table__cell">
+                  <a href="/BCST2/pathway/{{ pathway | getEnumByName }}/page/{{ questionAnswer.originalPageId }}?prisonerNumber={{ prisonerNumber }}&edit=true">Change</a>
```

Comment:
if it gets more complex than this it may be worth generating the url in an utils or controller so it can be tested